cli/src/tests/query_test.rs

```diff
@@ -4494,20 +4494,23 @@ fn test_query_max_start_depth() {
     #[rustfmt::skip]
     let rows = &[
         Row {
-            description: "depth 0: match all",
+            description: "depth 0: match translation unit",
             depth: 0,
             pattern: r#"
-                (if_statement) @capture
+                (translation_unit) @capture
             "#,
             matches: &[
-                (0, &[("capture", "if (a1 && a2) {\n    if (b1 && b2) { }\n    if (c) { }\n}")]),
-                (0, &[("capture", "if (b1 && b2) { }")]),
-                (0, &[("capture", "if (c) { }")]),
-                (0, &[("capture", "if (d) {\n    if (e1 && e2) { }\n    if (f) { }\n}")]),
-                (0, &[("capture", "if (e1 && e2) { }")]),
-                (0, &[("capture", "if (f) { }")]),
+                (0, &[("capture", "if (a1 && a2) {\n    if (b1 && b2) { }\n    if (c) { }\n}\nif (d) {\n    if (e1 && e2) { }\n    if (f) { }\n}\n")]),
             ]
         },
+        Row {
+            description: "depth 0: match none",
+            depth: 0,
+            pattern: r#"
+                (if_statement) @capture
+            "#,
+            matches: &[]
```

Comment:
This test is wrong. If 0 means top level depth then this should match 2 if statements

Reply:
Tto be clear, that test is from the commit prior to d4d5e29: https://github.com/tree-sitter/tree-sitter/pull/2140/commits/1e81a1b67f5aa8b17899186142a26eb422886065#diff-9743236475619a2b80048c89f911f576646f2050ea9d2c7120021d08895cfb75R4494-R4499

Are you saying it was wrong then?

Also, since the translation unit is at depth 0, wouldn't it make sense that no if statements would be matched at that depth?